feat(apple): Rename master to main branch (#6083)
getsentry/sentry-docs: src/platforms/apple/common/migration.mdx

## Migrating From 7.x to 8.x

> Migrating to 8.x from 7.x includes a few breaking changes. We provide this guide to help you to update your SDK. It is important to note that this version adds a dependency to Swift, and we renamed the default branch on the [sentry-cocoa repository](https://github.com/getsentry/sentry-cocoa) from `master` to `main`. We are going to keep the `master` branch for backwards compatibility for package managers not pointing to the `master` branch.
> 

Reviewer: ```suggestion
Migrating to 8.x from 7.x includes a few breaking changes. We provide this guide to help you update your SDK. It's important to note that this version adds a dependency to Swift, and that we renamed the default branch on the [sentry-cocoa repository](https://github.com/getsentry/sentry-cocoa) from `master` to `main`. We're keeping the `master` branch to ensure backwards compatibility for package managers that aren't pointing to the `master` branch.

```
